hepcrawl/pipelines.py

```diff
@@ -176,7 +176,7 @@ def __init__(self):
 
     def open_spider(self, spider):
         self.celery.conf.update(dict(
-            BROKER_URL=spider.settings['BROKER_URL'],
+            CELERY_BROKER_URL=spider.settings['CELERY_BROKER_URL'],
```

Comment:
does this require reverting https://gitlab.cern.ch/ai/it-puppet-hostgroup-inspire/commit/bb030ed9dea6a231cdedde4d8aa1349f92e91425 ?

Reply:
Eventually yes, although I will also need to fix the typo I made in https://gitlab.cern.ch/ai/it-puppet-hostgroup-inspire/commit/bb030ed9dea6a231cdedde4d8aa1349f92e91425#ece1b7b92ff70b0371eddaad468a2895e0fc8bb7_260_260.